pkgs/by-name/ap/apple-sdk/common/split-man-pages-output.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+}:
+
+self: super:
+let
+  # Tricky: fetchSDK cannot be called with boot fetchurl
+  isBoot = stdenv.name != "stdenv-darwin";
+
+  passthru = super.src.passthru or { };
+  includeMan = passthru ? includeMan && passthru.includeMan;
+in
+if !isBoot && includeMan then
```

Comment:
I apologize for not documenting this. There’s a blank `README.md` that I haven’t gotten back to writing yet. The SDK is built out of a series of overlays. The intent is they do what they do unconditionally with `package.nix` determining which overlays are applied. This condition should be moved there.

Reply:
Ok, I moved all the conditionals to `package.nix`. Having to check for boot fetchurl is ugly, too. I moved the `lib.toInt (lib.versions.major sdkVersion) >= 15` logic to `package.nix` so we don't need to look into `src.passthru`.